### DISCUSSION TITLE
setup magento pwa on windows local machine.

### DISCUSSION THREAD

**Discussion by amitmaurya1024** [magento/pwa-studio#3423] [Q&A]
Hello Masters,

I'm trying to install the PWA on my localhost machine. During installation I've faced lot of issues and fixed those issue now issue is when I run **yarn run watch:venia** it gets run with 100% then gives this error and PWA hosted URL. This error coming on both ( browser and terminal 
 I'm getting the following issue on browser

```
Module parse failed: Bad character escape sequence (4:79)
File was processed with these loaders:
* ../../node_modules/webpack-inject-plugin/dist/webpack-inject-plugin.loader.js
You may need an additional loader to handle the result of these loaders.
| return async function getLocale(locale) {
| if (locale === "en_US") {
> return import(/* webpackChunkName: "i18n-en_US" */'C:\xampp\htdocs\newpwa2\pwa-studio\packages\venia-concept\en_US.json');
| }
|
```

**Comment by Ramishscrapper** [comment]
@amitmaurya1024 

maybe this article will help you 

https://mage2.pro/t/topic/6230

but I will prefer these options to use WSL / docker / Virtual Machine / Dual boot it

**Comment by Chypro** [comment]
How did you solve the issue?
